docs/en/reference/information_schema/character_sets.md

```diff
@@ -0,0 +1,16 @@
+---
+displayed_sidebar: "English"
+---
+
+# character_sets
+
+`character_sets` identifies the character sets available.
+
+The following fields are provided in `character_sets`:
+
+| **Field**            | **Description**                                              |
+| -------------------- | ------------------------------------------------------------ |
+| CHARACTER_SET_NAME   | The character set name.                                      |
+| DEFAULT_COLLATE_NAME | The default collation for the character set.                 |
+| DESCRIPTION          | A description of the character set.                          |
+| MAXLEN               | The maximum number of bytes required to store one character. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8b08ed73-d6e9-4958-9992-cd0db3d0c39c)
should be "allowed"?

Reply:
required